dvc/plot.py

```diff
@@ -0,0 +1,232 @@
+import json
+import logging
+import os
+import random
+import re
+import string
+
+from funcy import cached_property
+
+from dvc.exceptions import DvcException
+from dvc.utils.fs import makedirs
+
+
+logger = logging.getLogger(__name__)
+
+
+class TemplateNotFound(DvcException):
+    def __init__(self, path):
+        super().__init__("Template: '{}' not found.".format(path))
```

Comment:
```suggestion
        super().__init__("Template '{}' not found.".format(path))
```
what's up with colons? Why do you like them so much? :smile: